src/System.Diagnostics.FileVersionInfo/src/System/Diagnostics/FileVersionInfo.Unix.cs

```diff
@@ -43,7 +43,7 @@ private bool TryLoadManagedAssemblyMetadata()
             try
             {
                 // Try to load the file using the managed metadata reader
-                using (FileStream assemblyStream = File.OpenRead(_fileName))
+                using (FileStream assemblyStream = new FileStream(_fileName, FileMode.Open, FileAccess.Read, FileShare.Read, bufferSize: 0x1000, useAsync: false))
```

Comment:
It's the default used if you don't specify a buffer size.  There's no ctor that let's you specify the async value without also specifying the buffer size, though, and there's no constant exposed that matches what's used internally, so I had to hardcode it.